MediaBrowser.Providers/MediaInfo/VideoImageProvider.cs

```diff
@@ -21,56 +21,55 @@ public class VideoImageProvider : IDynamicImageProvider, IHasOrder
     {
         private readonly IMediaEncoder _mediaEncoder;
         private readonly ILogger<VideoImageProvider> _logger;
-        private readonly IFileSystem _fileSystem;
 
-        public VideoImageProvider(IMediaEncoder mediaEncoder, ILogger<VideoImageProvider> logger, IFileSystem fileSystem)
+        public VideoImageProvider(IMediaEncoder mediaEncoder, ILogger<VideoImageProvider> logger)
         {
             _mediaEncoder = mediaEncoder;
             _logger = logger;
-            _fileSystem = fileSystem;
         }
 
+        /// <inheritdoc />
         public string Name => "Screen Grabber";
 
+        /// <inheritdoc />
         // Make sure this comes after internet image providers
         public int Order => 100;
 
+        /// <inheritdoc />
         public IEnumerable<ImageType> GetSupportedImages(BaseItem item)
         {
-            return new List<ImageType> { ImageType.Primary };
+            return new[] { ImageType.Primary };
```

Comment:
A micro-optimization for sure, but should we make a static "backing" field for this? Seems like a total waste to create new arrays all the time. And the caller of `GetSupportedImages` typically calls `.ToArray` on it as well :(